sos/report/plugins/vectordev.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Vectordev(Plugin):
+
+    short_desc = 'A lightweight, ultra-fast tool for building observability pipelines'
+
+    plugin_name = "vectordev"
+    profiles = ('observability',)
+
+    conf_file = "/etc/vector/vector.yaml"
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/vector/"
+        ])
+
+
+class UbuntuVectordev(Vectordev, UbuntuPlugin):
+    conf_file = "/etc/vector/aggregator/vector.yaml"
```

Comment:
Same as above. Is this necessary if you are capturing the whole /etc/vector?

Reply:
Er, I confused `conf_file` with `files` :facepalm: 

Good point, should I rewrite this as an `IndependentPlugin` ?

For the time being we just need to capture /etc/vector/ config files and nothing else, we can always change it in the future for distro specific. WDYT?

Reply:
> Another question - are there any containers or packages we should watch for? Like we do with microshift, i.e.:

Not for our specific use case, as we only get a binary and create the configuration files on `/etc/vector/`

Reply:
Re `IndependentPlugin`: per https://vector.dev/download/ , multiple OSes are supported, so this makes sense.